tests/primer/primer_tool.py

```diff
@@ -130,6 +132,22 @@ def _handle_run_command(self) -> None:
         ) as f:
             json.dump(packages, f)
 
+        # Fail loudly (and fail CI pipelines) if any fatal errors are found,
+        # unless they are astroid-errors, in which case just warn.
+        # This is to avoid introducing a dependency on bleeding-edge astroid
+        # for pylint CI pipelines generally, even though we want to use astroid main
+        # for the purpose of diffing emitted messages and generating PR comments.
+        messages = list(chain.from_iterable(packages.values()))
+        astroid_errors = [msg for msg in messages if msg["symbol"] == "astroid-error"]
+        other_fatal_msgs = [
+            msg
+            for msg in messages
+            if msg["type"] == "fatal" and msg["symbol"] != "astroid-error"
+        ]
+        if astroid_errors:
+            warnings.warn(f"Fatal errors traced to astroid:\n{astroid_errors}")
```

Comment:
Does this automatically translate into an Actions warning? (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-warning-message)

If not, that would be nice I guess.

Reply:
Maybe if we print something if would work like ``echo "::warning file=app.js,line=1,col=5,endColumn=7::Missing semicolon"`` from the example ?